src/uicommon.ml

```diff
@@ -494,10 +494,9 @@ let calcETA rem rate =
   (* Estimating the remaining time is not accurate. Reduce the display
      precision (and reduce more when longer time remaining). *)
   if t >= 86220 then
-    let d, h, (m, sec) =
-      let u = t + 180 in u / 86400, (u mod 86400) / 3600, (((u mod 3600) / 300) * 5, 0)
-    in
-    Printf.sprintf "%dd %02d:%02d:%02d" d h m sec
+    let u = t + 1799 in
+    Printf.sprintf "%dd %02d:%02d:00" (u / 86400) ((u mod 86400) / 3600)
```

Comment:
I am really not a fan of displaying a :00 for minutes, if I follow that. 

I am ok with us deciding that these strings are only for humans.  (If they are for computers, they should just be in seconds, the SI base unit :-)    If so, we could adopt a policy that if there are days, we should only days and hours, and if hours, only hours and minutes, and so on.   But I think showing a 0 in a field when it is not, is not ok.

Overall I lean to simpler, in code but especially in presentation to humans.

Reply:
(It's for the seconds in this case.)

I'm not sure I agree with you on this one. It sounds like form over function. How do you avoid ambiguity? You'd have to switch away from positional display then, right?

But this discussion should not extend the scope of this PR. I think this discussion instead goes well together with a discussion of the precision of time display (also separate from this PR).

Reply:
> showing a 0 in a field when it is not, is not ok.

For seconds this is what is already happening for any ETA above 2 minutes. Between 1–2 minutes ETA is rounded to the next 30 seconds, and below a minute the exact number of seconds is shown. The only change regarding rounding is that above 24 hours we further reduce precision from 5 to 30 minutes.

I don’t have a clear position on whether precision for ETAs above 24 h should be 5 m or 30 m. I’d suggest merging this PR for now – I’ll leave it up to you guys whether or not to make any changes to precision for ETA > 24 h before merging.

If there is a need to discuss what rounding we should apply when, or whether to display seconds when they are always zero, I would suggest doing so in a separate ticket. For seconds my only requirement is that I don’t want any ambiguity between `hh:mm` and `mm:ss`, even when I just have a single, static value such as a single log line.